app/scripts/controllers/reviewRegistration.js

```diff
@@ -279,7 +279,12 @@ angular.module('confRegistrationWebApp')
         // Reload the merged spouse registration to update the registration cost values before submitting payment
         return registration.load($scope.spouseRegistration.id);
       }).then(function (mergedRegistration) {
-        return payment.pay($scope.currentPayment, mergedRegistration, $scope.acceptedPaymentMethods());
+        return payment.pay($scope.currentPayment, mergedRegistration, $scope.acceptedPaymentMethods()).catch(function () {
+          // Payment errors do not stop the promise chain so that the page will still be updated with the merged registration
+          $scope.mergeError = 'An error occurred while processing your payment. ' +
```

Comment:
Can this go in the template to be translated? This could just be a flag to indicate to show that alert or not. I don't really think strings need to be in the JS. Angular templates handle them better.

As it is it isn't translated correctly.